ngraph/frontend/frontend_manager/CMakeLists.txt

```diff
@@ -33,3 +33,21 @@ target_include_directories(${TARGET_NAME} PUBLIC $<BUILD_INTERFACE:${FRONTEND_IN
 target_include_directories(${TARGET_NAME} PRIVATE ${NGRAPH_INCLUDE_PATH} ${FRONTEND_INCLUDE_DIR})
 
 target_include_directories(${TARGET_NAME} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/src)
+
+# Installation rules
+install(TARGETS ${TARGET_NAME} EXPORT ngraphTargets
+        RUNTIME DESTINATION ${NGRAPH_INSTALL_LIB} COMPONENT ngraph
+        ARCHIVE DESTINATION ${NGRAPH_INSTALL_LIB} COMPONENT ngraph
+        LIBRARY DESTINATION ${NGRAPH_INSTALL_LIB} COMPONENT ngraph)
+
+install(DIRECTORY ${FRONTEND_INCLUDE_DIR}/frontend_manager
+    DESTINATION ${FRONTEND_INSTALL_INCLUDE}
+    COMPONENT ngraph
+    FILES_MATCHING
+        PATTERN "*.hpp"
+        PATTERN "*.h"
+)
+
+if (NGRAPH_EXPORT_TARGETS_ENABLE)
```

Comment:
I think we can remove this useless option

Reply:
@ilya-lavrenov could you please clarify this? Everywhere else in ngraph (ngraph, onnx, protobuf) usage of 'export' is wrapped with this 'NGRAPH_EXPORT_TARGETS_ENABLE' option. Maybe this option can be turned off for some builds

Reply:
We have a task to remove `NGRAPH_EXPORT_TARGETS_ENABLE` because if we turn this option off and install artifacts, IE will depend on ngraph, but ngraph does not have config files / exported targets.

So, no needs to put this stuff under any options.

Reply:
ok, removed usage of this option